### PR TITLE
needed to add old script in to run, moved bad_address boolean to flag…

### PR DESCRIPTION
**Pull request by PJart**
Needed to add in old script to run production new preview geocoder updates while new script is tested

### CODE REVIEW

**Comment by mrblanco1** [comment]
Hello! Can someone please help us reviewing this and approve at your earliest convinience? Thank you!

File: lib/tasks/old_coord_mismatch.rake [hunk 1]
```diff
@@ -1,27 +1,53 @@
...
+      country.each do |result|
+        if result.state.present? && result.physical_country.present? && version.geocoded == false
+          geocoded_ct = Geocoder.coordinates(result.physical_country)
+          update_mismatch(result, geocoded_ct)
+        else
+          address = parse_address(result, result.address)
+          address2 = parse_address(result, result.address_2)
+          address3 = parse_address(result, result.address_3)
+          geocode_fields(result, address, address2, address3)
+        end
```
Comment (jhwuz) [inline lib/tasks/old_coord_mismatch.rake:32]:
```suggestion
        else
          addresses = [ parse_address(result, result.address),
                        parse_address(result, result.address_2),
                        parse_address(result, result.address_3) ]
          geocode_fields(result, addresses)
        end
```

File: lib/tasks/old_coord_mismatch.rake [hunk 2]
```diff
@@ -1,27 +1,53 @@
...
+  geocoded2 = Geocoder.coordinates(address2)
+  geocoded3 = Geocoder.coordinates(address3)
+  if geocoded.present?
+    update_mismatch(result, geocoded)
+  elsif geocoded2.present?
+    update_mismatch(result, geocoded2)
+  else
+    update_mismatch(result, geocoded3)
+  end
 end
```
Comment (jhwuz) [inline lib/tasks/old_coord_mismatch.rake:51]:
```suggestion
def geocode_fields(result, addresses)
  addresses.each do |address|
    geocoded = Geocoder.coordinates(address)
    update_mismatch(result, geocoded) if geocoded.present?
  end
end
```

File: lib/tasks/old_coord_mismatch.rake [hunk 3]
```diff
@@ -1,27 +1,53 @@
...
-        update_mismatch(result, geocoded2)
-      else
-        update_mismatch(result, geocoded3)
+  version = Version.current_preview
+  results = Institution.approved_institutions(version)
+  by_address = results.where(latitude: nil, longitude: nil).where.not(address_1: nil, city: nil)
+  country = results.where.not(physical_country: 'USA')
+  # will remove staging check, don't want it running on prod until tested
+  if version.present?
+    if by_address.present? && version.geocoded == false
```
Comment (jhwuz) [inline lib/tasks/old_coord_mismatch.rake:13]:
```suggestion
    if by_address.present? && !version.geocoded
```

File: lib/tasks/old_coord_mismatch.rake (hunk 2 above)
Comment (PJart) [reply lib/tasks/old_coord_mismatch.rake:51]: This is just an old script that is already tested and be taken out once we run, I haven't run this before so I don't want to make changes. This is just to do an update

**Review by jeremyhwu — APPROVED** [review]

**Pull request merged**